pkg/services/install/install.go

```diff
@@ -57,7 +59,25 @@ func (i *install) Install(namespace string, configURL gitproviders.RepoURL, auto
 		return fmt.Errorf("failed installing flux: %w", err)
 	}
 
-	manifests, err := models.BootstrapManifests(i.fluxClient, clusterName, namespace, configURL)
+	fluxNamespace, err := i.kubeClient.FetchNamespaceWithLabel(ctx, flux.PartOfLabelKey, flux.PartOfLabelValue)
```

Comment:
This block from line 62 to 80 seems duplicate from install/cmd.go. I wonder if there is a way we could avoid it. Maybe by passing in the kube client as parameter to `BoostrapManifests` ?